schema/schema.py

```diff
@@ -145,7 +145,10 @@ def format_param(param: click.core.Option) -> SamCliParameterSchema:
         items="string" if formatted_param_type == "array" else None,
     )
 
-    if param.default:
+    params_to_exclude_default = [
```

Comment:
Would it be possible to centralize this special parameter handling somewhere so that we don't need to have snippets of custom handling everywhere?

Not blocking on this since its something we can change later if we encounter more cases/need to refactor.

Reply:
Yup that's a good point, keeping everything centralized will definitely help 👍